packages/client-utils/src/signing-smart-wallet-kit.ts

```diff
@@ -59,7 +59,7 @@ type WalletStoreEntryProxy<T, Recursive extends true | false = false> = {
 
 // TODO parameterize as part of https://github.com/Agoric/agoric-sdk/issues/5912
 const defaultFee: StdFee = {
-  amount: [{ denom: 'ubld', amount: '500000' }], // XXX enough?
+  amount: [{ denom: 'ubld', amount: '15000' }], // XXX enough?
```

Comment:
This PR should only lower it to an amount that is enough.

```suggestion
  amount: [{ denom: 'ubld', amount: '15000' }],
```